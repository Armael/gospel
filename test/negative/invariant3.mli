type t = int * int (*@ invariant 1 > 0 *)
