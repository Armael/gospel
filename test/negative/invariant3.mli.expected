
*******************************
********** Parsed file ********
*******************************
type t = (int * int)[@@gospel {| invariant 1 > 0 |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Stdlib *)

(*@ open Gospelstdlib *)

type t = (int * int)
  (*@ invariant ...
       *)
File "invariant3.mli", line 1, characters 0-51:
Error: Invariant on public type `t'.
